src/main/java/mate/academy/internetshop/dao/jdbc/UserDaoJdbcImpl.java

```diff
@@ -0,0 +1,172 @@
+            }
+            addRolesForUser(user);
+            return user;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Creation user is failed", e);
+        }
+    }
+
+    @Override
+    public Optional<User> get(Long id) {
+        String query = "SELECT * FROM internet_shop.users WHERE user_id=?";
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement statement = connection.prepareStatement(query);
+            statement.setLong(1, id);
+            ResultSet resultSet = statement.executeQuery();
+            if (resultSet.next()) {
+                return Optional.of(getUserFromResultSet(resultSet));
+            }
+            return Optional.empty();
+        } catch (SQLException e) {
+            throw new DataProcessingException("User is not exist", e);
+        }
+    }
+
+    @Override
+    public List<User> getAll() {
+        String query = "SELECT * FROM internet_shop.users";
+        List<User> usersList = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement statement = connection.prepareStatement(query);
+            ResultSet resultSet = statement.executeQuery();
+            while (resultSet.next()) {
+                usersList.add(getUserFromResultSet(resultSet));
+            }
+            return usersList;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't take users list", e);
+        }
+    }
+
+    @Override
+    public User update(User user) {
+        String query = "UPDATE internet_shop.users "
+                + "SET name=?, login=?, password=? WHERE user_id=?;";
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement statement = connection.prepareStatement(query);
+            statement.setString(1, user.getName());
+            statement.setString(2, user.getLogin());
+            statement.setString(3, user.getPassword());
+            statement.setLong(4, user.getId());
+            statement.executeUpdate();
+            deleteRolesFromUser(user);
+            addRolesForUser(user);
+            return user;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't update user", e);
+        }
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String query = "DELETE FROM internet_shop.users WHERE user_id=?;";
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement statement = connection.prepareStatement(query);
+            statement.setLong(1, id);
+            statement.executeUpdate();
+            return true;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't delete user", e);
+        }
+    }
+
+    private void addRolesForUser(User user) {
+        String query = "INSERT INTO internet_shop.users_roles VALUES (?,?);";
```

Comment:
```suggestion
        String query = "INSERT INTO users_roles VALUES (?,?);";
```
пробіжися по всьому коду, пліз